compliance/repository/src/test/java/org/eclipse/rdf4j/repository/sparql/federation/RepositoryFederatedServiceIntegrationTest.java

```diff
@@ -129,7 +129,7 @@ public void test4() throws Exception {
 				vf.createStatement(iri("s2"), RDFS.LABEL, l("val2")),
 				vf.createStatement(iri("s3"), RDFS.LABEL, l("val3"))));
 
-		String query = "SELECT ?var WHERE { SERVICE <urn:dummy> { ?s ?p ?var } . SERVICE <urn:dummy> {  ?s ?p ?var  } }";
+		String query = "SELECT ?var WHERE { SERVICE <urn:dummy> { ?s ?p ?var } . SERVICE <urn:dummy> {  ?s ?p ?var  } } order by ?var";
```

Comment:
There were tests that assumed an order but didn't specify one. The new ConcurrentHashSet based model has reverse order from the LinkedHashSet based one. I think this is fine, unless there is some internal ordering requirement that I don't know about???

Reply:
A Model is supposed to have predictable iteration order but that doesn't translate to a fixed order on a SPARQL query result, so good find.